x-pack/plugins/monitoring/server/config.ts

```diff
@@ -57,7 +57,6 @@ export const configSchema = schema.object({
     }),
   }),
   cluster_alerts: schema.object({
-    enabled: schema.boolean({ defaultValue: true }),
```

Comment:
I think this would be a breaking change in a minor. You should double check, but I think if you define configs that do not exist, Kibana will not start.

Reply:
Yepp, you're right! 

I will add it back